lib/challenge.js

```diff
@@ -164,11 +206,17 @@ const starWars = ["Han", "C3PO", "R2D2", "Luke", "Leia", "Anakin"];
 
 function prompt9() {
```

Comment:
For hits, you want to loop through the `starWars` array, not just manually access each item inside the array using `[sumNumberHere]`. Imagine if you had 100 items in the array! So a better solution would be a loop. If the instructions say `iterate` that means `loop`. I would try that out and let me know how that goes!